src/shell/js/composeui-fdc3/src/ComposeUITopic.ts

```diff
@@ -0,0 +1,25 @@
+/* 
+ *  Morgan Stanley makes this available to you under the Apache License,
+ *  Version 2.0 (the "License"). You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0.
+ *  See the NOTICE file distributed with this work for additional information
+ *  regarding copyright ownership. Unless required by applicable law or agreed
+ *  to in writing, software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ *  or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ *  
+ */
+
+export class ComposeUITopic {
+    private static readonly topicRoot = "composeui/fdc3/v2.0/";
+    private static readonly userChannelTopicRoot: string = this.topicRoot + "userchannels/";
+
+    static broadcast(topicId: string) {
+        return this.userChannelTopicRoot + topicId + "/broadcast";
```

Comment:
Isn't broadcast channel-specific? Shouldn't `broadcast` and `subscribe` return the same topic for a given `channelId` and `contextType`?